keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -65,7 +65,7 @@
             }
           },
           "default": {
-            "description": "Default response. It will be deserialized as per the error definition specified in the schema. An exception will be thrown.",
+            "description": "CDN error response describing why the operation failed.",
```

Comment:
@BrucePerlerMS and I discussed this later in email, and speculated the same (ie: this was a copy/paste error). I've noticed this in another (non-CDN) JSON spec as well.